README.md

```diff
@@ -1,6 +1,10 @@
+[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/comit-network/community)
```

Comment:
I thought we said Slack?

Reply:
Rediscussed it with Philipp, in order to avoid spam Gitter will serve as first contact for now. Slack invites are handed out after first contact.